examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -2745,11 +2742,10 @@ void loop(void) {
         if (i < capture.rawlen - 1)
           lastIrReceived += ',';
       }
-    }
 #endif  // REPORT_RAW_UNKNOWNS
     // If it isn't an AC code, add the bits.
     if (!hasACState(capture.decode_type))
-      lastIrReceived += kCommandDelimiter[0] + String(capture.bits);
+      lastIrReceived += resultToHexidecimal(&capture);
```

Comment:
Do you think we could submit this code upstream? Or transform it in a way we could submit to the official project?

Reply:
(I let a comment below)